core/src/components/item/item.tsx

```diff
@@ -128,20 +128,30 @@ export class Item implements ComponentInterface, AnchorInterface, ButtonInterfac
   }
 
   componentDidLoad() {
-    // Check for multiple inputs to change the position to relative
-    const inputs = this.el.querySelectorAll('ion-select, ion-datetime');
-    this.multipleInputs = inputs.length > 1 ? true : false;
+    // The following elements have a clickable cover that is relative to the entire item
+    const covers = this.el.querySelectorAll('ion-checkbox, ion-datetime, ion-select, ion-radio');
+
+    // The following elements can accept focus alongside the previous elements
+    // therefore if these elements are also a child of item, we don't want the
+    // input cover on top of those interfering with their clicks
+    const inputs = this.el.querySelectorAll('ion-input, ion-range, ion-textarea, ion-toggle');
```

Comment:
We might need `ion-searchbar` here
```suggestion
    const inputs = this.el.querySelectorAll('ion-input, ion-range, ion-textarea, ion-toggle, ion-searchbar');
```

Reply:
also noticing this breaks if a user uses `ion-segment`. It's not technically an input, but maybe we can put a check for it somewhere?